exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/using_a_script_languages_container.ipynb

```diff
@@ -0,0 +1,704 @@
+    "\n",
+    "#### Embedded Build Information of the Container\n",
+    "Here we see an overview about the build information which exaslct embedded into the container. `exaslct` stores all packages lists (as defined in the flavor and what actually got installed), the final Dockerfiles and the image info. The image info describes how the underlying Docker images of the container got built. The build information is stored in the `/build_info` directory in the container. We can use again our helper UDF to inspect the build information."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "92b4f6d4-9235-4e51-a3f4-1ee73aae90ef",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rs=conn.execute(\"\"\"select execute_shell_command_py3('find /build_info', true)\"\"\")\n",
+    "for r in rs: \n",
+    "    print(r[0])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b3b2810b-ad74-47bb-a94f-34f6cb9e459a",
+   "metadata": {},
+   "source": [
+    "The following command shows for example, which python3 package pip found directly after the build of the container image."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4f716612-fa6e-496f-9ec9-39fefb647b60",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rs=conn.execute(\"\"\"select execute_shell_command_py3('cat /build_info/actual_installed_packages/release/python3_pip_packages', true)\"\"\")\n",
+    "for r in rs: \n",
+    "    print(r[0])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "42e6e17c-ff3c-4311-a0c9-b044b8d7d6a9",
+   "metadata": {},
+   "source": [
+    "You could for example compare this to the package list of the flavor-customization build step and check if all your requested packages got installed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "aefe0c99-32d5-4357-bc2e-1131c7b38b9a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rs=conn.execute(\"\"\"select execute_shell_command_py3('cat /build_info/packages/flavor_customization/python3_pip_packages', true)\"\"\")\n",
+    "for r in rs:\n",
+    "    if r[0] is None:\n",
+    "        print()\n",
+    "    else:\n",
+    "        print(r[0])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a0528564-fe3d-4143-8b58-6473ce250c60",
+   "metadata": {},
+   "source": [
+    "### Testing the new package\n",
+    "\n",
+    "After we made sure that the required packages are installed, we need to try importing and using it. Importing is usually a good first test if a package got successfully installed, because often you might already get errors at this step. However, sometimes you only will recognize errors when using the package. We recommend to have a test suite for each new package to check if it works properly before you start your UDF development. It is usually easier to debug problems if you have very narrow tests."
```

Comment:
```suggestion
    "After we made sure that the required packages are installed, we need to try importing and using them. Importing is usually a good first test if a package got successfully installed, because often you might already get errors at this step. However, sometimes you only will recognize errors when using the package. We recommend to have a test suite for each new package to check if it works properly before you start your UDF development. It is usually easier to debug problems if you have very narrow tests."
```

Reply:
Fixed